shuup/admin/templates/shuup/admin/base_picotable.jinja

```diff
@@ -15,4 +15,31 @@
 {% endblock %}
 {% block extra_js %}
 <script src="{{ shuup_static("shuup_admin/js/picotable.js") }}"></script>
+{% if listview %}
```

Comment:
this script shouldn't be here, but in the a custom template, like here: `shuup/simple_supplier/templates/shuup/simple_supplier/admin/base_picotable.jinja`